apps/vibe-coding-platform/ai/tools/create-sandbox.description.ts

```diff
@@ -1,4 +1,4 @@
-Use this tool to create a new Vercel Sandbox — an ephemeral, isolated Linux container that serves as your development environment for the current session. This sandbox provides a secure workspace where you can upload files, install dependencies, run commands, start development servers, and preview web apps. Each sandbox is uniquely identified and must be referenced for all subsequent operations (e.g., file generation, command execution, or URL access).
+export default `Use this tool to create a new Vercel Sandbox — an ephemeral, isolated Linux container that serves as your development environment for the current session. This sandbox provides a secure workspace where you can upload files, install dependencies, run commands, start development servers, and preview web apps. Each sandbox is uniquely identified and must be referenced for all subsequent operations (e.g., file generation, command execution, or URL access).
```

Comment:
Sadly I had to change these from `md` files since workflow devkit doesn't currently let you provide your own loaders to the workflow bundling interface

Hope this isn't a blocker. We'd like to support custom loaders in devkit but that isn't high priority